pkg/cmd/run/cancel/cancel.go

```diff
@@ -56,6 +57,8 @@ func NewCmdCancel(f *cmdutil.Factory, runF func(*CancelOptions) error) *cobra.Co
 		},
 	}
 
+	cmd.Flags().BoolVarP(&opts.Force, "force", "f", false, "Force cancel a workflow run when it doesn't respond to regular cancel")
```

Comment:
Suggestion: I think this should be less prescriptive about when to use it. 🙏 

```suggestion
	cmd.Flags().BoolVarP(&opts.Force, "force", "f", false, "Force cancel a workflow run")
```